### PR TITLE
Films 2016

### PR DESCRIPTION
**Pull request by davojta**

### CODE REVIEW

**Comment by davojta** [comment]
should be merge after https://github.com/kinaklub/next.filmfest.by/pull/63

**Comment by davojta** [comment]
need to fix incorrect translation pickup for missed ones

File: .gitignore [hunk 1]
```diff
@@ -6,3 +6,17 @@ test_db.sqlite3
...
 .tox
 .cache
+
+.idea/
+.ipynb_checkpoints/
+next.filmfest.by.iml
+*.ipynb
+./*.json
+
+scripts/films.json_to_next_raw_data/node_modules/*
```
Comment (staaas) [inline .gitignore:16]: this seems to be duplicated in #66

**Comment by davojta** [comment]
will be included in https://github.com/kinaklub/next.filmfest.by/pull/66

**Pull request closed without merging**